packages/react-ds/src/components/Select/Select.stories.mdx

```diff
@@ -6,7 +6,7 @@ import Spinner from '../Spinner/Spinner'
 import Docs from '@venice/core/docs/Select.md'
 
 import Select from './Select'
-import StoreIcon from '../icons/Store'
+import StoreIcon from '../Icons/Store'
 
 <Meta title="Components|Selects" component={Select} />
```

Comment:
Maybe we can create a new category to Forms?

```
<Meta title="Forms|Selects" component={Select} />
```

Reply:
Then we move Input and Select for there??